gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dcheckboxlabel_ttt2.lua

```diff
@@ -63,6 +63,13 @@ function PANEL:Paint(w, h)
 	return true
 end
 
+---
+-- @param bool invert
+-- @realm client
+function PANEL:SetInverted(invert)
+	self.inverted = invert
+end
+
```

Comment:
I feel like this only adds unnecessary complexity. What value does this provide? Which use cases do you have?

Reply:
As said, we currently have old values in use like notbuyable, that we changed to buyable and are therefore inverted.
Until we have some #673 to track changes and maybe force some, I would leave it like this.

Reply:
I agree with Sven here. We decided a while ago to have no inverted variables. To achieve that we either have to drop the current settings, use migrations or set an inverted flag

Reply:
i think that is a fair reasoning. But i would like that reasoning to be put into the function documentation, so that anyone who comes across this can understand why it was done.